docs/cards.html

```diff
@@ -0,0 +1,369 @@
+        active="cards"
+      ></custom-sidebar>
+      <main class="flex-grow-1 pb-16 pi-16 spacer">
+        <h1>Cards</h1>
+        <p>
+          Cards are built with as little markup and styles as possible, but
+          still manage to deliver a ton of control and customization. Built with
+          flexbox, they offer easy alignment and mix well with other Finesse UI
+          components. They have no margin by default, so use spacing utilities
+          as needed. Cards have no fixed width to start, so they’ll naturally
+          fill the full width of its parent element. This is easily customized
+          with the sizing of your choice.
+        </p>
+        <hr />
+
+        <h2>Text Only Card</h2>
+        <p>
+          Below is an example of a <strong>basic text only card</strong>. Add a
+          <code>.card.card-potrait.card-text-only</code> class to turn a div
+          into a card and add <code>.card-description-action</code> class as
+          child element which can hold all your text.
+        </p>
+        <div class="panel">
+          <div class="code-preview">
+            <div class="card-container-demo">
+              <div class="card card-potrait card-text-only">
+                <div class="card-description-action">
+                  <div class="card-description mb-8 mi-8">
+                    <h2 class="h4">Card Heading</h2>
+                    <h2 class="h6">Card Sub Heading</h2>
+                    <p>A card description with only text as example</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="code-container">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Source+Code+Pro&fs=18px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card%2520card-potrait%2520card-text-only%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-description-action%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-description%2520mb-8%2520mi-8%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Ch2%2520class%253D%2522h4%2522%253ECard%2520Heading%253C%252Fh2%253E%250A%2520%2520%2520%2520%2520%2520%253Ch2%2520class%253D%2522h6%2522%253ECard%2520Sub%2520Heading%253C%252Fh2%253E%250A%2520%2520%2520%2520%2520%2520%253Cp%253EA%2520card%2520description%2520with%2520only%2520text%2520as%2520example%253C%252Fp%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+              style="
+                width: 637px;
+                height: 272px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
```

Comment:
avoid inline styling

Reply:
bro this is coming from the carbon, sh its self, each has different width based on the code i pasted

Reply:
fixed